quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/logging/KeycloakLogFilter.java

```diff
@@ -55,6 +63,21 @@ public boolean isLoggable(LogRecord record) {
             }
         }
 
+        if (Thread.currentThread().getName().startsWith("non-blocking-thread")) {
```

Comment:
Most virtual threads do not have names, and JGroups vthreads do not start with `non-blocking-thread`. 
You probably want to use `org.infinispan.commons.jdkspecific.ThreadCreator#isVirtual(Thread)`

Reply:
Thank you, I was hoping someone would have implemented something like that to make the detection available independent of the JDK version! Applied in next commit - ready for one more review.